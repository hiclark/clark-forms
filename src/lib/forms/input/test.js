import React from 'react';

import Input from './';

describe('<Input />', () => {
  let wrapper;
  beforeEach(() => {
    const props = {
      name: '',
      label: '',
      copy: '',
      inputType: 'text',
      required: true,
      disabled: false,
      tooltip: null,
      hasShowHideButton: false,
      stripped: true,
    };
    wrapper = shallow(<Input {...props} />);
  });

  it('renders correctly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
