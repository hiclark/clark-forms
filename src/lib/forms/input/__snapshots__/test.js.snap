// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Input /> renders correctly 1`] = `
<React.Fragment>
  <Field
    component={[Function]}
    copy=""
    disabled={false}
    hasShowHideButton={false}
    inputType="text"
    label=""
    name=""
    required={true}
    stripped={true}
    toggleInputVisibility={[Function]}
    tooltip={null}
  />
</React.Fragment>
`;
